src/server/services/validate-upload.js

```diff
@@ -43,6 +43,13 @@ async function validateEcCode ({ upload, records, trns }) {
   const coverSheet = records.find(doc => doc.type === 'cover').content
   const codeString = coverSheet['Detailed Expenditure Category']
 
+  if (codeString === undefined) {
+    return new ValidationError(
+      'Record is missing EC code',
```

Comment:
not sure if `"Record"` is the right term here. maybe `"Upload"` ? also, maybe something like `"EC code must be set"` work match the missing-agency error well?